huxley/core/models.py

```diff
@@ -306,7 +306,7 @@ def remove(assignment_data):
             deletions.append(assignment_data['id'])
 
         for committee, country, school, rejected in new_assignments:
-            key = (committee, country)
+            key = (committee.id, country.id)
```

Comment:
brutal. was this causing all the assignments to be deleted? Why didn't our tests catch this?